robot_manipulation/PackItem[] items  # in order
